src/hid/hid-device.cpp

```diff
@@ -60,11 +60,21 @@ namespace librealsense
             _sensor_to_id[gyro] = REPORT_ID_GYROMETER_3D;
             _sensor_to_id[accel] = REPORT_ID_ACCELEROMETER_3D;
             _sensor_to_id[custom] = REPORT_ID_CUSTOM;
+#ifdef __APPLE__
+            hidapi_device_info *devices = hid_enumerate(0x8086, 0x0);
+            if(devices)
+            {
+                _hidapi_device = hid_open(devices[0].vendor_id, devices[0].product_id, devices[0].serial_number);
+                hidapi_PowerDevice(REPORT_ID_ACCELEROMETER_3D);
+                hidapi_PowerDevice(REPORT_ID_GYROMETER_3D);
+            }
+#endif
             _action_dispatcher.start();
         }
 
         rs_hid_device::~rs_hid_device()
         {
+            hid_close(_hidapi_device);
```

Comment:
This should be surrounded by `#ifdef __APPLE__`.
Also, check for `nullptr`, please, since it's not always initialized?

Reply:
Corrected